packages/material-ui/src/Stepper/Stepper.test.js

```diff
@@ -176,6 +173,25 @@ describe('<Stepper />', () => {
       const connectors = wrapper.find(StepConnector);
       expect(connectors.first().props().completed).to.equal(true);
     });
+
+    it('should pass correct active and completed props to the StepConnector with nonLinear prop', () => {
+      const steps = ['Step1', 'Step2', 'Step3'];
+
+      const wrapper = mount(
+        <Stepper orientation="horizontal" nonLinear connector={<StepConnector />}>
+          {steps.map((label, index) => (
+            <Step key={label} active completed={index === 2}>
+              <StepLabel>{label}</StepLabel>
+            </Step>
+          ))}
+        </Stepper>,
+      );
+
+      expect(wrapper.find(StepConnector).at(0).props().active).to.equal(true);
```

Comment:
Please test this with the CSS API, also, could you use testing library instead of enzyme for the next tests?

Reply:
Sure, I'll do it tonight!

Reply:
Already fixed :)

Reply:
@oliviertassinari Thanks! Are you planning to rewrite all existing tests to testing library? It looks like it's a good possibility for future contributions :)

Reply:
Yes, we are rewriting existing tests to testing-library. It would make a great future contributions :)